cmd/fyne/internal/commands/package-unix.go

```diff
@@ -67,9 +68,11 @@ func (p *Packager) packageUNIX() error {
 			return fmt.Errorf("failed to write Makefile string: %w", err)
 		}
 
+		buf := bytes.Buffer{}
```

Comment:
nit: I don't think it makes a difference, but I think looks neater
```suggestion
		var buf bytes.Buffer
```